recipes/mpir/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "2.7.2" %}
+
+package:
+  name: mpir
+  version: {{ version }}
+
+source:
+  fn: mpir-{{ version }}.tar.bz2
+  url: http://mpir.org/mpir-{{ version }}.tar.bz2
+  md5: b31aa1ebc01b5fedf4ac43277c03feec
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - m4  # [unix]
+    - python
+    - vs2015_runtime # [win]
+  run:
+    - vs2015_runtime # [win]
+
+test:
+  files:
```

Comment:
To explain a little bit more, MPIR needs Python at its build phase to run some python scripts for building. (Any python version >= 2.4 would do). It's not required at runtime.